Makefile

```diff
@@ -139,23 +149,56 @@ OBJS += \
 	pow_fpga_accel.o
 endif
 
+ifeq ("$(BUILD_REMOTE)", "1")
+OBJS += \
+	remote_common.o \
+        remote_interface.o
+
+WORKER_OBJS := $(addprefix $(OUT)/remote/,$(filter-out remote_interface.o, $(OBJS)))
+OBJS := $(addprefix $(OUT)/, $(OBJS))
+LIBDCURL_OBJS := $(OBJS)
+OBJS += $(WORKER_OBJS)
```

Comment:
While the `BUILD_REMOTE` is equal to 1, the variable `OBJS` is not used in the following rules.
Is the variable `LIBDCURL_OBJS` really necessary?

Reply:
OBJS is for building total objects including LIBDCURL_OBJS and WORKER_OBJS. LIBDCURL_OBJS is for linking them for libdcurl.so. WORKER_OBJS is for linking them for remote-worker.